src/backend/crossterm.rs

```diff
@@ -16,6 +16,10 @@ impl CrosstermBackend {
         }
     }
 
+    pub fn with_crossterm(scr: crossterm::Screen) -> CrosstermBackend {
```

Comment:
To be a bit more consistent with the actual parameter, what about:
```suggestion
    pub fn with_screen(screen: crossterm::Screen) -> CrosstermBackend {
```

Reply:
Sounds good.